apptentive/src/com/apptentive/android/sdk/module/engagement/interaction/view/TextModalInteractionView.java

```diff
@@ -107,11 +116,14 @@ public void onClick(View view) {
 								if (interactionIdToLaunch != null) {
 									Interactions interactions = InteractionManager.getInteractions(activity);
 									if (interactions != null) {
-										Interaction interaction = interactions.getInteraction(interactionIdToLaunch);
+										Interaction invokedInteraction = interactions.getInteraction(interactionIdToLaunch);
 										if (interaction != null) {
```

Comment:
`interactionToInvoke` or `interactionToLaunch` I would think... it isn't "invoked" until the end.